kahuna/public/js/search/results.js

```diff
@@ -196,7 +196,8 @@ results.controller('SearchResultsCtrl', [
                 until:      until || $stateParams.until,
                 since:      since || $stateParams.since,
                 offset:     offset,
-                length:     length
+                length:     length,
+                orderBy:    $stateParams.orderBy
```

Comment:
Assuming this will be undefined at this point and defaulted on the server?
Do we want to be explicit for the app? Or do we want the default set by the server?